### DISCUSSION TITLE
Installing v1.10.9 \\ Site showing up but /dashboard doesn't work

### DISCUSSION THREAD

**Discussion by derkolarsky** [automadcms/discussions#26] [Q&A] [answered]
Hello dear friends,

I have a problem that I didn't get solved by now. I'm experimenting a little with a small local Ubuntu Server (based on a Mac Mini Server). System works fine so far but when installing Automad I run into problems…
- the system is set up with Apache
- I set chown -r to www-data:www-data and chmod -r 775 for the `automad` directory
- installed PHP8.3 and 8.1 as well. But switching between the version doesn't make a difference
- the system is set up to ssl (LE) and is forwarding to Port 443 as wanted
- when opening the URL in a browser: `…/automad` I see the regular page, but `…/automad/dashboard` gives me an error 404

I'm not a pro in Linux so maybe I just missed to set up something important.

Maybe one of you folks has a clever idea for me…?

Thanks in advance
Ralf

**Comment by marcantondahmen** [comment, marked as answer]
Hi there! 

It looks like url rewriting doesn't work correctly. Is the rewrite module enabled in your Apache config?

Also version 1.10.9 requires PHP between versions 7.4 and 8.1, PHP 8.2+ is not supported.

Regarding permissions, the entire installation directory should be writable by PHP, not only the `automad` directory.

**Comment by derkolarsky** [comment]
Sometimes solutions are so simple… Haven't seen this one. Now it's fine. Thanks a lot @marcantondahmen for your quick response. Appreciate it. 👼

**Reply by marcantondahmen** [reply]
You're welcome.
